src/game_state/game.py

```diff
@@ -30,23 +36,50 @@ def __init__(self):
             )
         )
         self.score = 0
-        self.fruit_spawn = True
 
-    def touch_boundaries(self) -> bool:
+    def _touch_boundaries(self) -> bool:
+        """Checks whether the snake's head is touching the
+        boundaries of the board.
+
+        Returns:
+            bool: True in case the snake's head is touching the boundaries, False otherwise.
+        """
```

Comment:
This is something I have thought of but never mentioned. IDK if the comments should be in Spanish, not too sure but maybe yes. What you think?

Reply:
They should be in Spanish, yes.

Reply:
But I don't know if the code details should be in Spanish also